util/rlp/fuzz/fuzz_targets/untrusted.rs

```diff
@@ -0,0 +1,11 @@
+#![no_main]
+#![cfg(feature = "nightly")]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rlp;
+
+use rlp::UntrustedRlp;
+
+fuzz_target!(|data: &[u8]| {
+    let _ = UntrustedRlp::new(data);
```

Comment:
I added another fuzz test that creates a list (prepending `0xc + data_len` to the fuzz data).

> probably this fuzz test should try and iterate over all lists recursively.

Do you mean creating lists of various types from the fuzz data, and recursively calling `UntrustedRlp::new(fuzz-list-data)` on those lists?